content/300-accelerate/600-faq.mdx

```diff
@@ -96,4 +96,20 @@ Even without using Accelerate's global cache, you can still greatly benefit from
 
 ## Can I use Accelerate with other ORMs/query builders/drivers?
 
-No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to reach out and let us know in our [Discord](https://pris.ly/discord) community in the `#accelerate-feedback` channel.
+No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to reach out and let us know in our [Discord](https://pris.ly/discord) community in the `#help-and-questions` channel.
+
+## What is the maximum allowable value for the `ttl` parameter when configuring cacheStrategy?
+
+The [Time-to-live (`ttl`)](/accelerate/caching#time-to-live-ttl) parameter can be set for up to a **year**. However, it's important to note that items within the cache may be evicted if they are not frequently accessed.
+
+Based on our experimentation, we’ve seen cache items persist for around 18 hours. While items may remain in the cache for an extended period if they are actively accessed, there is no guarantee.
+
+> Even frequently accessed items may occasionally be evicted from the cache. It's unlikely for an item to survive for up to or longer than a month, regardless of its activity level.
```

Comment:
being consistent with the other block quote and prepending the block quote with **Note**: 

```suggestion
> **Note**: Even frequently accessed items may occasionally be evicted from the cache. It's unlikely for an item to survive for up to or longer than a month, regardless of its activity level.
```